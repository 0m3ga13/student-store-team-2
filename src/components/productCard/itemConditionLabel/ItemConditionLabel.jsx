import Image from "next/image";
import React from "react";
import { useTranslations } from "next-intl";

const ItemConditionLabel = ({ itemCondition }) => {
    const t = useTranslations("Index");

    return (
        <div className='relative'>
            <Image
                className='z-0'
                src='/conditionLabelVector.svg'
                alt='Item Condition'
                width={139}
                height={39}
            ></Image>
            <div className='font-jost font-extrabold text-sm text-bkg absolute bottom-1 left-2'>
                {t("State")}: {itemCondition}
            </div>
        </div>
    );
};

export default ItemConditionLabel;
