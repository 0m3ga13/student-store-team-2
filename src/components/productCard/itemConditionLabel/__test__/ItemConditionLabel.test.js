import renderer from "react-test-renderer";

import ItemConditionLabel from "../ItemConditionLabel";

it("renders correctly", () => {
    const tree = renderer.create(<ItemConditionLabel />).toJSON();
    expect(tree).toMatchSnapshot();
});
