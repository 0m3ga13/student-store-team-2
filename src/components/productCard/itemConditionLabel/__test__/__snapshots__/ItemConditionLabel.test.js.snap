// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="relative"
>
  <img
    alt="Item Condition"
    className="z-0 text-blue-400"
    data-nimg="1"
    decoding="async"
    height={39}
    loading="lazy"
    onError={[Function]}
    onLoad={[Function]}
    src="/conditionLabelVector.svg"
    style={
      Object {
        "color": "transparent",
      }
    }
    width={139}
  />
  <div
    className="font-jost font-extrabold text-sm text-bkg absolute bottom-1 left-2 w-fit"
  >
    $
  </div>
</div>
`;
